### ISSUE TITLE
Study view: Responsive tiles

### ISSUE THREAD

**Issue by kerzmaximilian** [RADAR-CNS/RADAR-Dashboard#226]
- [x] Tile's height adjust to view height.

- [ ] Depending on device width, adjust tile layout to a stacked column view.

**Comment by herkulano** [comment]
@kerzmaximilian 

> Depending on device width, adjust tile layout to a stacked column view.

We've discussed the idea of having the dashboard on mobile and tablet devices and we've postponed this because it needs to be well thought out. The dashboard needs to be completely redesigned for these sizes. This includes thinking not only how it will look on a smaller device, ie., font sizes, heights, etc, but also what information to put on a smaller screen and how it will be presented. For example, something as simple as the data table in the Study View will have several functional and usability problems if we simply "squeeze" it into a mobile or tablet view.
